src/table/table-adapter.class.ts

```diff
@@ -181,4 +263,66 @@ export class TableDomAdapter implements TableAdapter {
 	findIndex(cell: HTMLTableCellElement): [number, number] {
 		return [this.findRowIndex(cell), this.findColumnIndex(cell)];
 	}
+
+	/**
+	 * Helper function that returns the "real" length of a row.
+	 * Only accurate with regard to colspans (though that's sufficient for it's uses here)
+	 *
+	 * TODO: Take rowSpan into account
+	 *
+	 * @param row the row to get the length of
+	 */
+	protected getRealRowLength(row: HTMLTableRowElement): number {
+		return Array.from(row.cells).reduce((count, cell) => {
+			return count + cell.colSpan;
```

Comment:
can we drop the `return` here?

Reply:
Yeah, so long as it doesn't make the line unreadable